includes/fraud-prevention/class-fraud-risk-tools.php

```diff
@@ -170,7 +170,7 @@ public static function get_standard_protection_settings() {
 				Check::check(
 					'order_total',
 					Check::OPERATOR_GT,
-					1000 * 100
+					implode( '|', [ 1000 * 100, 'usd' ] )
```

Comment:
When I activated the high protection level, and went to the advanced config, I saw this: 
<img width="606" alt="image" src="https://user-images.githubusercontent.com/3295/229096527-c5984b50-98d7-481c-966b-1fa246907c0f.png">

Probably we should convert 1000$ into the store currency and write it here?

Reply:
That's a good idea! I started to convert the value on the rulesets default value. It was addressed on [2deffc6](https://github.com/Automattic/woocommerce-payments/pull/5926/commits/2deffc6786e11e09449955197b698a19e4c4c6d7).